README.md

```diff
@@ -5,70 +5,58 @@
 
-Available on `Python` 3.7-3.12.
-- [`comtypes==1.2.1`](https://pypi.org/project/comtypes/1.2.1/) is the last version supporting `Python` 2.7 and 3.3-3.6.
-- In `Python` 3.7.6 and 3.8.1, `comtypes` would not work as reported in [GH-202](https://github.com/enthought/comtypes/issues/202).  
-This bug has been fixed in `Python` >= 3.7.7 and >= 3.8.2.
-- In `Python` 3.8 and 3.9, some of `comtypes` functionalities may not work correctly as reported in [GH-212](https://github.com/enthought/comtypes/issues/212).  
-This bug has been fixed in `Python` >= 3.10.10 and >= 3.11.2.
-
-## Where to get it
-
-The source code is currently hosted on GitHub at:
-https://github.com/enthought/comtypes
-
-An installer for the latest released version is available at the [Python Package Index (PyPI)](https://pypi.org/project/comtypes).
+## Installation
 
+`comtypes` is available on [PyPI](https://pypi.org/project/comtypes) and may be installed with `pip`:
 ```sh
 # PyPI
 pip install comtypes
 ```
 
-## Dependencies
-
-`comtypes` requires no third-party packages to run - this is truly **pure** `Python` package.
+The source code for `comtypes` may be found on [GitHub](https://github.com/enthought/comtypes).
 
-Optional features include the follows...
-- to process arrays as `numpy`'s `ndarray`
-- type hints be interpreted by `mypy` or several static type checkers
+### Dependencies
 
-But these third-parties are not required as a prerequisite for runtime.
+`comtypes` is a pure Python package — it has no additional required dependencies.
 
-## Community of the developers
+Optional functionality may be enabled by installing:
+- `numpy`, in order to process arrays as `numpy`'s `ndarray`.
+- `mypy` or other static type checkers, in order to interpret type hints.
 
-Tracking issues, reporting bugs and contributing to the codebase and documentation are on GitHub at:
-https://github.com/enthought/comtypes
+None of these packages, however, are required in order to run `comtypes`.
 
-## For Enterprise
+## Community
 
-Available as part of the Tidelift Subscription.
+Issues may be tracked, bugs may be reported and contributions may be made to the codebase and documentation on `comtypes`' [GitHub](https://github.com/enthought/comtypes) repository.
 
-This project and the maintainers of thousands of other packages are working with Tidelift to deliver one enterprise subscription that covers all of the open source you use.
+## Enterprise
 
-[Learn more](https://tidelift.com/subscription/pkg/pypi-comtypes?utm_source=pypi-comtypes&utm_medium=referral&utm_campaign=github).
+`comtypes` is available as part of the Tidelift Subscription. `comtypes` and the maintainers of thousands of other packages are working with Tidelift to deliver one enterprise subscription that covers all of the open source you use. To learn more, you can visit Tidelift's website [here](https://tidelift.com/subscription/pkg/pypi-comtypes?utm_source=pypi-comtypes&utm_medium=referral&utm_campaign=github).
```

Comment:
```suggestion
## For Enterprise

Available as part of the Tidelift Subscription.

This project and the maintainers of thousands of other packages are working with Tidelift to deliver one enterprise subscription that covers all of the open source you use.

[Learn more](https://tidelift.com/subscription/pkg/pypi-comtypes?utm_source=pypi-comtypes&utm_medium=referral&utm_campaign=github).
```

This section is quoted from [jaraco/tidelift](https://github.com/jaraco/tidelift).  
I think it is understandable as it is.

Reply:
Fair enough.